src/game/server/gamecontext.cpp

```diff
@@ -129,28 +129,109 @@ class CCharacter *CGameContext::GetPlayerChar(int ClientID)
 	return m_apPlayers[ClientID]->GetCharacter();
 }
 
-int CGameContext::GetZombieCount() {
-	int count = 0;
+int CGameContext::GetActivePlayerCount(){
+	
+	// returns how many players are currently playing and not spectating
+		int PlayerCount = 0;
+		int SpecCount = 0;
+		for(int i=0; i<MAX_CLIENTS; i++)
+		{
+			//COMPILER ERROR ‘class IServer’ has no member named ‘m_aClients’
+			if(Server()->m_aClients[i].m_State == CServer::CClient::STATE_INGAME)
```

Comment:
I think its best to create GetActivePlayerCount and GetSpectatorCount inside server.cpp
the array m_aClients will only be used inside server.cpp